nixos/modules/services/networking/powerdns.nix

```diff
@@ -0,0 +1,65 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.powerdns;
+  configDir = pkgs.writeTextDir "pdns.conf" "${cfg.extraConfig}";
+in {
+  options = {
+    services.powerdns = {
+      enable = mkOption {
+        default = false;
+        description = "Whether to enable Powerdns domain name server.";
+      };
+
+      package = mkOption {
+        default = pkgs.powerdns;
+        defaultText = "pkgs.powerdns";
+        description = "Which powerdns derivation to use";
```

Comment:
Missing a dot here
Also is it really necessary to make the package configurable? it's only necessary if there is more pdns pkg in nixpkgs and you want to run multiple instances of the service with different pkgs